src/lib/appManagers/appStateManager.ts

```diff
@@ -147,156 +139,188 @@ export const STATE_INIT: State = {
       sound: false
     }
   },
-  keepSigned: true,
-  drafts: {}
+  keepSigned: true
 };
 
 const ALL_KEYS = Object.keys(STATE_INIT) as any as Array<keyof State>;
 
 const REFRESH_KEYS = ['dialogs', 'allDialogsLoaded', 'messages', 'contactsList', 'stateCreatedTime',
-  'updates', 'maxSeenMsgId', 'filters', 'topPeers'] as any as Array<keyof State>;
+  'updates', 'maxSeenMsgId', 'filters', 'topPeers', 'pinnedOrders'] as any as Array<keyof State>;
 
 export class AppStateManager extends EventListenerBase<{
   save: (state: State) => Promise<void>,
   peerNeeded: (peerId: number) => void,
   peerUnneeded: (peerId: number) => void,
 }> {
   public static STATE_INIT = STATE_INIT;
-  public loaded: Promise<State>;
+  private loaded: Promise<State>;
   private log = logger('STATE'/* , LogLevels.error */);
 
   private state: State;
-  private savePromise: Promise<void>;
-  private tempId = 0;
 
   private neededPeers: Map<number, Set<string>> = new Map();
   private singlePeerMap: Map<string, number> = new Map();
 
+  public storages = {
+    users: new AppStorage<Record<number, User>>({
+      storeName: 'users'
+    }),
+
+    chats: new AppStorage<Record<number, Chat>>({
+      storeName: 'chats'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/MikeOwino/tweb/pullRequest?prid=7329411)
```suggestion
      storeName: "chats"
```